writing-apps/the-basic-setup.md

```diff
@@ -90,10 +90,23 @@ sudo apt install elementary-sdk
 
 On elementary OS 6 beta and newer, you should already have the required Flatpak remote and platform pre-installed. On earlier versions or other OSes, you can add the remote and install the Flatpak platform and SDK:
 
-```bash
-flatpak remote-add --if-not-exists --system appcenter https://flatpak.elementary.io/repo.flatpakrepo
-flatpak install -y appcenter io.elementary.Platform io.elementary.Sdk
-```
+{% tabs %}
+    {% tab title="OS 8 and Above" %}
+
+    {% endtab %}
+
+    {% tab title="OS 6 and Above" %}
+        ```bash
+        flatpak install -y appcenter io.elementary.Sdk
+        ```
+    {% endtab %}
+{% tab title="Before OS 6" %}
```

Comment:
Anything before OS 6 is now EOL so we can just drop those instructions completely or you could make a tab for `elementary OS` and `Other Linux OS`

Reply:
I went with the second option :)